packages/mesa.rb

```diff
@@ -27,55 +19,40 @@ class Mesa < Package
   depends_on 'libxfixes'
   depends_on 'libxdamage'
   depends_on 'libxcb'
+  depends_on 'libxvmc'
   depends_on 'libxtrans'
   depends_on 'libxkbfile'
   depends_on 'wayland'
   depends_on 'python27'
   depends_on 'bison'
   depends_on 'libxshmfence'
   depends_on 'elfutils'
+  depends_on 'llvm'
   
   def self.build
-    #system "pip install --prefix #{CREW_PREFIX} --root #{CREW_DEST_DIR} Mako"
+    system "pip uninstall -y Mako MarkupSafe || :"
```

Comment:
Or colon?  What is the purpose of doing that?

Reply:
The command fails if `Mako` and `MarkupSafe` are not installed. `:` is an alias for `true`.

Reply:
This line won't be necessary if there is a `python_mako` package dependency.